client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -58,37 +64,36 @@ export default function PlanOnlyThankYou( {
 		wpcom.req.post( '/me/send-verification-email', ( error: Error ) => {
 			if ( error ) {
 				errorNotice( translate( "Couldn't resend verification email. Please try again." ), {
-					id: VERIFY_EMAIL_ERROR_NOTICE,
+					id: verifyEmailNoticeId,
+					duration: 5000,
 				} );
 
 				setResendStatus( RESEND_ERROR );
 				return;
 			}
 
+			successNotice( translate( 'Email sent' ), { id: verifyEmailNoticeId, duration: 5000 } );
+
 			setResendStatus( RESEND_SUCCESS );
 		} );
 	};
 
 	const resendButtonText = () => {
-		switch ( resendStatus ) {
-			case RESEND_PENDING:
-				return translate( 'Sending…' );
-			case RESEND_SUCCESS:
-				return translate( 'Email sent' );
-			case RESEND_NOT_SENT:
-			case RESEND_ERROR:
-			default:
-				return translate( 'Resend email' );
+		if ( resendStatus === RESEND_PENDING ) {
+			return translate( 'Sending…' );
 		}
+
+		return translate( 'Resend email' );
```

Comment:
I realize we copied this logic from the old Congrats page, but I figured we could improve the UX slightly as part of the redesign. 

I think it'd be a good idea to be consistent with how we notify users on the resend status. It felt strange that we show a notice when an error is encountered, but not when the email was sent successfully. From the code's perspective, I also find it redundant to define a bunch of switch cases that end up falling back to the default. So I've gone ahead and simplified the `resendButtonText()` function to a simple `if/else`. Instead of changing the button label to "Email sent" when an email was sent successfully, we fallback to the original label "Resend email", with the button disabled, indicating that user isn't expected to trigger another resend.